typed-racket-test/fail/apply-dots.rkt

```diff
@@ -1,13 +1,13 @@
 #;
-(exn-pred 3)
+(exn-pred 2)
```

Comment:
Why did this change?

Reply:
This was a silly mistake on my part. The file still errors w/ 3 type checker errors (I had changed the file, forgot I changed it, and mindlessly updated the exn-pred line (doh!))

It's fixed now. Same behavior as before